noir-projects/aztec-nr/aztec/src/keys/getters.nr

```diff
@@ -74,10 +110,29 @@ fn fetch_key_from_registry(
         AztecAddress::from_field(CANONICAL_KEY_REGISTRY_ADDRESS),
         y_coordinate_derived_slot
     );
-    let x_coordinate = x_coordinate_registry.get_current_value_in_private();
-    let y_coordinate = y_coordinate_registry.get_current_value_in_private();
 
-    GrumpkinPoint::new(x_coordinate, y_coordinate)
+    (x_coordinate_registry, y_coordinate_registry)
```

Comment:
The wording here makes it sound like it is separate registries, but it is just different coordinates so think I would still keep the naming similar to not make misunderstandings that it is a registry 🤷